File: PracticasProgramacion/Practica_13/python/coche.py
"""Coche: precio y año."""


class Coche:
    def __init__(self, precio: float = 0.0, anio: int = 0) -> None:
        self._precio = precio
        self._anio = anio

    def get_precio(self) -> float:
        return self._precio

    def get_anio(self) -> int:
        return self._anio

File: PracticasProgramacion/Practica_13/cpp/Coche.cpp
#include "Coche.h"

Coche::Coche() : precio_(0), anio_(0) {}
Coche::Coche(double precio, int anio) : precio_(precio), anio_(anio) {}
double Coche::getPrecio() const { return precio_; }
void Coche::setPrecio(double precio) { precio_ = precio; }
int Coche::getAnio() const { return anio_; }
void Coche::setAnio(int anio) { anio_ = anio; }
